### PR TITLE
INDY-1950: Rename TRUST_ANCHOR to ENDORSER

### PR DESCRIPTION
**Pull request by ArtObr**
Signed-off-by: ArtObr <artemobruchnikov@gmail.com>

### CODE REVIEW

**Comment by ArtObr** [comment]
test this please

File: acceptance/indy-cli-batches/AS-03-01-trustee.batch [hunk 1]
```diff
@@ -9,8 +9,8 @@ did new seed=TestTrustee200000000000000000000 metadata="test trustee 2"
 did new seed=TestTrustee300000000000000000000 metadata="test trustee 3"
 did new seed=TestSteward100000000000000000000 metadata="test steward 1"
 did new seed=TestSteward200000000000000000000 metadata="test steward 2"
-did new seed=TestTrustAnchor00000000000000000 metadata="test trust anchor 1"
-did new seed=TestTrustAnchor10000000000000000 metadata="test trust anchor 2"
+did new seed=TestEndorser00000000000000000 metadata="test endorser 1"
```
Comment (ashcherbakov) [inline acceptance/indy-cli-batches/AS-03-01-trustee.batch]: The seed's length needs to be 32. Please add missing 0s.

File: environment/vagrant/sandbox/DevelopmentEnvironment/AWS/scriptlets/indy-self-serve/linux/test [hunk 2]
```diff
@@ -7,4 +7,4 @@ echo "Testing lambda/nym ..."
 # Python wrappers require libindy and libindy requires libsodium                
 # Add /usr/local/lib to LD_LIBRARY_PATH for libsodium                           
 # Add /src/indy-sdk/libindy/target/debug to LD_LIBRARY_PATH for libindy         
-sudo -H -u ec2-user PYTHONPATH=$PYTHONPATH:/src/indy-sdk/wrappers/python LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib:/src/indy-sdk/libindy/target/debug lambda/nym/test/TrustAnchorAddSandboxTest.sh
+sudo -H -u ec2-user PYTHONPATH=$PYTHONPATH:/src/indy-sdk/wrappers/python LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib:/src/indy-sdk/libindy/target/debug lambda/nym/test/EndorserAddSandboxTest.sh
```
Comment (ashcherbakov) [inline environment/vagrant/sandbox/DevelopmentEnvironment/AWS/scriptlets/indy-self-serve/linux/test]: Was this file really renamed?

File: acceptance/indy-cli-batches/AS-03-01-trustee.batch (hunk 1 above)
Comment (ArtObr) [reply acceptance/indy-cli-batches/AS-03-01-trustee.batch]: fixed

File: environment/vagrant/sandbox/DevelopmentEnvironment/AWS/scriptlets/indy-self-serve/linux/test (hunk 2 above)
Comment (ArtObr) [reply environment/vagrant/sandbox/DevelopmentEnvironment/AWS/scriptlets/indy-self-serve/linux/test]: fixed

**Pull request merged**